.github/pr-title-checker-config.json

```diff
@@ -0,0 +1,17 @@
+{
+	"LABEL": {
+		"name": "title needs formatting",
+		"color": "EEEEEE"
+	},
+	"CHECKS": {
+		"prefixes": ["build: ", "chore: ", "ci: ", "docs: ", "feat: ", "fix: ", "perf: ", "refactor: ", "revert: ", "test: "],
```

Comment:
PR Title should have these prefixes and regex, which is working fine in Embedded app SDK.